src/vivarium_census_prl_synth_pop/model_specifications/model_spec.yaml

```diff
@@ -30,7 +30,7 @@ configuration:
         extrapolate: True
     randomness:
         map_size: 1_000_000
-        key_columns: ['entrance_time', 'age']
+        key_columns: []  # We do not utilize crn
```

Comment:
Is this because we throw everything into vectorized choice and not the normal functionality of CRN?

Reply:
No, it boils down to the INdexMap getter function https://github.com/ihmeuw/vivarium/blob/main/src/vivarium/framework/randomness/index_map.py#L243-L247.

I'm still trying to wrap my head around it, but basically if you don't properly register new simulants to the randomness stream (which we were not) and you have `key_columns`, then this getter tries to return self._map.loc[index].values except we don't have a map (b/c that's part of registering new simulants).

So we could have just registered new simulants to the randomness stream every time we create them, but that's unnecessary b/c we don't actually use crn. So instead, we "cheated" by just removing the `key_columns`.